src/api/emoji.rs

```diff
@@ -0,0 +1,38 @@
+//!
+//! Support for Slack Team API methods
+//!
+
+use std::collections::HashMap;
+
+use rsb_derive::Builder;
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+
+use crate::models::*;
+use crate::ratectl::*;
+use crate::SlackClientSession;
+use crate::{ClientResult, SlackClientHttpConnector};
+
+impl<'a, SCHC> SlackClientSession<'a, SCHC>
+where
+    SCHC: SlackClientHttpConnector + Send,
+{
+    ///
+    /// https://api.slack.com/methods/emoji.list
+    ///
+    pub async fn emoji_list(&self) -> ClientResult<SlackApiEmojiListResponse> {
+        self.http_session_api
+            .http_get(
+                "emoji.list",
+                &crate::client::SLACK_HTTP_EMPTY_GET_PARAMS.clone(),
+                Some(&SLACK_TIER2_METHOD_CONFIG),
+            )
+            .await
+    }
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackApiEmojiListResponse {
+    pub emoji: HashMap<String, String>,
```

Comment:
Would be great to reuse existing type for key `SlackEmoji` to indicate what we expect.

Also we can introduce one more type something like `SlackEmojiRef` or similar name  for values. In this case people would now what are they about (self-documenting), and also we can introduce methods on it to convert to Url etc in future.

Hope those comments make sense, I like strong typing :)